<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tea_addscore</class>
 <widget class="QDialog" name="tea_addscore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>学生选课和成绩管理系统</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加成绩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addscore_add">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addscore_cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>110</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_addscore_name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>学号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_addscore_id">
        <property name="maxLength">
         <number>9</number>
        </property>
        <property name="placeholderText">
         <string>请输入9位学号</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>班级：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_addscore_class">
        <property name="maxLength">
         <number>6</number>
        </property>
        <property name="placeholderText">
         <string>请输入6位班号</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>数学：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_addscore_math">
        <property name="maxLength">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>英语：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_addscore_eng">
        <property name="maxLength">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>物理：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_addscore_phy">
        <property name="maxLength">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
